gen/aircraft.py

```diff
@@ -1162,6 +1158,15 @@ def build(self) -> MovingPoint:
         self.set_waypoint_tot(waypoint, self.timing.ingress)
         return waypoint
 
+class CasIngressBuilder(PydcsWaypointBuilder):
+    def build(self) -> MovingPoint:
+        waypoint = super().build()
+        self.set_waypoint_tot(waypoint, self.timing.ingress)
+        waypoint.add_task(EngageTargets(max_distance=nm_to_meter(10),
+                              targets=[Targets.All.GroundUnits.GroundVehicles])
```

Comment:
This excludes infantry and AAA. Should probably include those.

Reply:
I added AAA to the engage task, but it causes erratic behavior from the AI.  I think they're stuck in a loop of trying to target a SAM, then going defensive, then targeting again.

I'm going to play with a few options here.

Reply:
If I had actually applied the change as you suggested, it would have worked fine :laughing: .  This latest commit 8583bbf has just AAA (not SAMs) and infantry added to the task, and resolves the wandering CAS flight issue.